<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibOfHomeworks</class>
 <widget class="QDialog" name="LibOfHomeworks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>筛选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="course_select"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>请在下表选择题目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="SelectAll">
         <property name="text">
          <string>全选</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="work_table"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addSelected">
     <property name="text">
      <string>加入</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>即将加入以下题目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="SelectAll_2">
         <property name="text">
          <string>全选</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="selected_table"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QRadioButton" name="setRelativeDate">
       <property name="text">
        <string>将截止日期延后一些天数</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="setNewDate">
       <property name="text">
        <string>设置新的截止日期</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDateEdit" name="dateset"/>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="datedelta">
       <property name="maximum">
        <number>800</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="applyDatesetting">
     <property name="text">
      <string>应用日期设置</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>添加到</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="course_dest"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="SaveWorks">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Exit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
